src/main/java/util/TransactionValidator.java

```diff
@@ -0,0 +1,15 @@
+package util;
+
+import model.FruitTransaction;
+
+public class TransactionValidator {
+    public static void validate(FruitTransaction transaction) {
```

Comment:
if it's a utility class, common convention is to add a private constructor to signal that no instantiation is supposed to happen for this class